connectors/uipath/src/main/java/io/camunda/connector/Main.java

```diff
@@ -0,0 +1,5 @@
+package io.camunda.connector;public class Main {
+    public static void main(String[] args) {
+        System.out.println("Hello world!");
+    }
+}
```

Comment:
We don't need it, you're right! I will delete it!

Reply:
I will close this PR because we will need to create a new version of the custom template. I will create a separate task for it. The new versions will have to support two types of request "Add queue item" and "Get item status". There will be conditions for every input field, to be displayed depending on the type of request selected.  We can consider the first draft created and the new version will be based on it.